src/main/java/org/folio/circulation/domain/representations/RequestByInstanceIdRequest.java

```diff
@@ -0,0 +1,133 @@
+package org.folio.circulation.domain.representations;
+
+import static org.folio.circulation.support.Result.succeeded;
+import static org.folio.circulation.domain.RequestFulfilmentPreference.NONE;
+import static org.folio.circulation.support.JsonPropertyFetcher.getDateTimeProperty;
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+import static org.folio.circulation.support.JsonPropertyFetcher.getUUIDProperty;
+import static org.folio.circulation.support.ValidationErrorFailure.failedValidation;
+
+import java.util.UUID;
+
+import org.folio.circulation.domain.RequestFulfilmentPreference;
+import org.folio.circulation.support.Result;
+import org.joda.time.DateTime;
+
+import io.vertx.core.json.JsonObject;
+
+public class RequestByInstanceIdRequest {
+  private static final String JSON_FIELD_REQUEST_DATE = "requestDate";
```

Comment:
Why do these use the `JSON_FIELD` prefix? Is it to try to discriminate them from the fields? 

Could that be achieved a different way?

Reply:
Sorry, the prefix is just habit. In cases where there are a bunch a string constants, I ten to use a prefix to clue developers what the intended use of the string is without having to consult the code. I was debating this here, since all string constants are names of JSON fields. I'll just remove the prefix.

Reply:
Fair enough. We all have different habits like that.

I'd likely choose to move them to a separate class, to group them together (however Java / SonarQube make that more expensive than I'd like). And maybe not use static imports to keep the context.